WordPress/Classes/Utility/Blogging Reminders/BloggingReminders.swift

```diff
@@ -0,0 +1,157 @@
+import Foundation
+
+/// Main interface for scheduling blogging reminders
+///
+class BloggingReminders {
```

Comment:
Since this is the type that schedules, maybe we can call it `BloggingReminderScheduler`?

Reply:
I considered it but I was wondering if we could ever want to have more in here than just the scheduling.  Any thoughts on this?

If we feel this is unlikely, I'm good with the name `BloggingRemindersScheduler`.

Reply:
It really depends. If it works (and aside from the name), maybe we can start considering a smaller set of responsibilities for this class and expand it (and rename it) if necessary?

Reply:
Sounds good to me.

Reply:
Changed the name of the class to `BloggingRemindersScheduler`.